src/app/components/profile/profile.component.scss

```diff
@@ -1,82 +1,88 @@
 @import './variables.scss';
 @import './mixins.scss';
+ 
+#card_container {
+	
+.card {
+	background: $gray-lighter;
+	padding-bottom:30px;
+	width:50%;
+	margin:0 auto;
```

Comment:
I think you are missing here. Please give one tab space to these(6 to 10 line number) lines.